### PR TITLE
display: Switch "Open in Browser" to "View in Browser"

### PR DESCRIPTION
**Pull request by abhinav**
Switch this to "View in Browser" per discussion
which is closer to the old "View Live" message.

sdk/go/auto doesn't need an update because the regex there
is still using "View in Browser".

### CODE REVIEW

File: pkg/backend/display/display.go [hunk 1]
```diff
@@ -40,10 +40,10 @@ func printViewLive(out io.Writer, opts Options, permalink string) {
 	printPermalink(out, opts, "View Live", permalink)
 }
 
-// printOpenInBrowser print's an update's permalink prefaced with `Open in Browser (Ctrl+O): `. This is printed in
+// printOpenInBrowser print's an update's permalink prefaced with `View in Browser (Ctrl+O): `. This is printed in
 // interactive scenarios that use the tree renderer.
 func printOpenInBrowser(term terminal.Terminal, opts Options, permalink string) {
```
Comment (pgavlin) [inline pkg/backend/display/display.go:45]: might as well rename the function too:

```suggestion
func printViewInBrowser(term terminal.Terminal, opts Options, permalink string) {
```

Comment (abhinav) [reply pkg/backend/display/display.go:45]: Done

File: pkg/backend/display/display.go [hunk 2]
```diff
@@ -40,10 +40,10 @@ func printViewLive(out io.Writer, opts Options, permalink string) {
 	printPermalink(out, opts, "View Live", permalink)
 }
 
-// printOpenInBrowser print's an update's permalink prefaced with `Open in Browser (Ctrl+O): `. This is printed in
-// interactive scenarios that use the tree renderer.
-func printOpenInBrowser(term terminal.Terminal, opts Options, permalink string) {
```
Comment (dixler) [inline pkg/backend/display/display.go:45]: I'd almost name these `printPermalinkInteractive` and `printPermalinkNonInteractive`

Comment (abhinav) [reply pkg/backend/display/display.go:45]: Renamed

**Comment by abhinav** [comment]
bors merge

**Pull request merged**